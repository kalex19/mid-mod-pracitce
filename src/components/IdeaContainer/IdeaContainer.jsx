import React from 'react';
import './IdeaContainer.css';
import IdeaCard from '../IdeaCard/IdeaCard';

export default function IdeaContainer({ ideas, deleteIdeas }){
	const ideaCard = ideas.map(idea => <IdeaCard deleteIdeas={deleteIdeas} {...idea} key={idea.id} />);
	return <div>{ideaCard}</div>;
}
