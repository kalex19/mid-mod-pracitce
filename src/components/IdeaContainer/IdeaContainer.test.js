import React from 'react'
import {
  shallow
} from 'enzyme';
import IdeaContainer from './IdeaContainer';

describe('IdeaContainer', () => {
      let wrapper;
      let mockIdeas;
      let mockDeleteIdeas;

      beforeEach = () => {
        mockIdeas = [{
          title: 'Hello',
          body: 'World'
        }, {
          title: 'Goodbye',
          body: 'moon'
        }]
        mockDeleteIdeas = jest.fn()
        wrapper = shallow( < Container ideas = {
            mockIdeas
          }
          deleteIdeas = {
            mockDeleteIdeas
          }
          />)
        }

        it('should match snapshot', () => {
          expect(wrapper).toMatchSnapshot()
        })
      })