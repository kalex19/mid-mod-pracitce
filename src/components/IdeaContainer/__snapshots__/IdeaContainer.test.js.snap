// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdeaContainer should match snapshot 1`] = `undefined`;
